src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -112,6 +112,10 @@ public static class TextLoaderSaverCatalog
             bool allowSparse = TextLoader.Defaults.AllowSparse)
         {
             Contracts.CheckNonEmpty(path, nameof(path));
+            if (!File.Exists(path))
```

Comment:
>!File.Exists(path) [](start = 16, length = 18)

Consider adding this check to the `BinaryLoader` and `SvmLightLoader` APIs.

Reply:
Good point, I just did!

Reply:
Good point, I just did!